### DISCUSSION TITLE
request add supported language korean

### DISCUSSION THREAD

**Discussion by kijung-iM** [oramasearch/orama#748] [Q&A]
Hello, 

I'm using docusaurus and I want to try the orama plugin. The plug-in doesn't support Korean yet, so I'm wondering if you have any plans to add it.

**Comment by micheleriva** [comment]
Hi @kijung-iM, Korean is a bit tricky, especially without a person who can read and understand it in the team. Let me see if I can find some help.

**Reply by kijung-iM** [reply]
please refer following link.

https://github.com/turbobit/lunr-languages/blob/master/lunr.ko.js

**Comment by korECM** [comment]
@micheleriva 
Hi, I'd like to follow up on this — is there any update on Korean language support?

I'm currently building a documentation site and would love to use Orama for search, but Korean support is a blocker for us. I noticed the discussion has been quiet for a while, so I wanted to check if this is still on the roadmap.

I'm a Korean-speaking developer and would be happy to help with testing, providing sample text/queries, or contributing to the implementation if there's a direction the team is considering. Please let me know if there's anything I can do to move this forward!

**Reply by micheleriva** [reply]
Hi @korECM, I tried implementing Korean, but sadly, I've never been able to pass tests. When compared to Japanese, Mandarin, Arabic, and Russian (talking about different alphabets here), I find it a bit more difficult to implement - I apologize for that.

**Reply by korECM** [reply]
@micheleriva 
Thanks for the context.
Since I’m a Korean speaker, I think I might be able to help, but I wanted to first understand where things got stuck.
I took a quick look at the codebase and didn’t see any existing Korean-specific test cases. Could you share how you set up and ran the tests when you tried implementing Korean support?

**Comment by Cyberheroes1010** [comment]
Biggest fruad Company !! Block me without paying my money !! Doing scam with researcher today i make a post on twitter & all social media !!

<img width="1701" height="835" alt="Image" src="https://github.com/user-attachments/assets/a9d5470d-00e0-4919-b64f-eebf35b2933e" />

<img width="1903" height="945" alt="Image" src="https://github.com/user-attachments/assets/e2f28ac4-967f-4f85-96db-da3a3d784413" />
<img width="1601" height="823" alt="Image" src="https://github.com/user-attachments/assets/071d318a-565d-4357-8257-7613b1d83c81" />

**Reply by Cyberheroes1010** [reply]
Biggest fruad Company !! Block me without paying my money !! Doing scam with researcher today i make a post on twitter & all social media !!

<img width="1701" height="835" alt="Image" src="https://github.com/user-attachments/assets/a9d5470d-00e0-4919-b64f-eebf35b2933e" />

<img width="1903" height="945" alt="Image" src="https://github.com/user-attachments/assets/e2f28ac4-967f-4f85-96db-da3a3d784413" />
<img width="1601" height="823" alt="Image" src="https://github.com/user-attachments/assets/071d318a-565d-4357-8257-7613b1d83c81" />
